reporter/src/main/kotlin/reporters/CycloneDxReporter.kt

```diff
@@ -107,7 +107,7 @@ class CycloneDxReporter : Reporter {
         val projects = input.ortResult.getProjects(omitExcluded = true)
         val createSingleBom = !options[OPTION_SINGLE_BOM].isFalse()
 
-        if (createSingleBom && projects.size > 1) {
+        if (createSingleBom) {
```

Comment:
I wonder whether we shouldn't do `createSingleBom || projects.size == 1` instead. What do you think?

Reply:
May be those who are "multi bom" want to have the project name in the file name even when there is only one project ?
I feel it's more consistence.

Reply:
I have no strong opinion here. Your proposal makes the code simpler, so I'm fine with it.

A few commit message nits, though:

> CycloneDxReporter: force the default name even for one project. 

Should be "CycloneDxReporter: Force the default name even for one project" (start capitalized after a colon, no dot at the end of the title).

> we can expect to have the same behavior with 1

Please avoid "we" or "I" and use passive language instead, like "it is reasonable to expect". Please spell out "1" as "one".